uploadservice/src/main/java/net/gotev/uploadservice/UploadNotificationConfig.java

```diff
@@ -218,11 +218,11 @@ public UploadNotificationStatusConfig getCancelled() {
         return cancelled;
     }
 
-    public String getHighImportanceNotificationChannelId() {
+    public String getMaxImportanceNotificationChannelId() {
         return highImportanceNotificationChannel.first;
     }
 
-    public String getHighImportanceNotificationChannelName() {
+    public String getMaxImportanceNotificationChannelName() {
```

Comment:
I think you should rename `highImportanceNotificationChannel` to `maxImportanceNotificationChannel` and also rename the setter method - `setHighImportanceNotificationChannel()`.